README.md

```diff
@@ -3,40 +3,79 @@
 
 # Sprocket GitHub Action
 
-This action uses [Sprocket](https://github.com/stjude-rust-labs/sprocket) to validate and optionally lint WDL documents.
+This action provides the functionality of the [Sprocket](https://github.com/stjude-rust-labs/sprocket) tool.
```

Comment:
This sentence feels kind of half-finished to me. I'd add a little more, something like:
```suggestion
This action provides select functionality of the [Sprocket](https://github.com/stjude-rust-labs/sprocket) command line tool for use in CI/CD pipelines.
```